frontend/src/utils/apiWrapper.js

```diff
@@ -25,12 +25,36 @@ export const getResponseByTractID = tract_id => {
     });
 };
 
-export const getResponseRatesByDate = date => {
-  const requestString = `${BASE_URL}response_rates?date=${date}`;
+export const getResponseRatesByYear = year => {
+  const requestString = `${BASE_URL}rate?year=${year}`;
```

Comment:
i don't think these do the same thing- aren't you supposed to change just `response_rates` to `rate`?